index.html

```diff
@@ -35,252 +36,92 @@
+              reaching new areas, new industries, and new users. Be a part of the growing Python community 
+              and help shape its future. You don't need to be a well traveled professional speaker to get on 
+              stage at Pycon Sweden. What matters is the topics, the materials, and the drive to present things 
+              to this community.
             </p>
+            <h3>Where and when does the conference take place?</h3>
             <p>
-              In order to make our conference even better, please
-              provide us feedback about your experience. That makes us
-              create even more amazing conference next year!
+              Pycon Sweden 2015 will take place in Stockholm in May. We will announce the exact dates and venues <b>February 2</b>, at the latest.
+            </p>
+            <h3>Do you cover speaker expenses?</h3>
+            <p>
+              Sorry, Pycon Sweden is a not-for-profit venture, so speakers are expected to pay for their own tickets, travel and accommodation.
+            </p>
+            <h3>When should I submit a proposal?</h3>
+            <p>
+              You can submit a proposal starting today until <b>February 16</b>, and we plan to respond within two weeks of that date.
+            </p>
+            <h3>How do I submit a proposal?</h3>
+            <p>
+              Submitting a proposal is easy, <a href="http://goo.gl/forms/V0tgG0pQ1G">just fill out this form</a>. Here are some great links from PyCon US on how to create a good proposal:
             </p>
-          </div>
-          <div class="col-sm-12">
-            <h2>Venue</h2>
-            <p>PyCon Sweden will be held at the <a href="http://www.kth.se" target="_new">KTH Royal Institute of Technology</a>, Q building (<a href="https://www.google.com/maps/place/Osquldas+väg+4/@59.3502295,18.0677362,17z" target="_new"><strong>map</strong></a>) in central Stockholm, close the the beautiful Haga park and Brunnsviken bay. Several hotels and hostels are available in the vicinity (<a href="https://www.google.com/maps/search/hotel/@59.3502295,18.0677362,15z" target="_new">hotel search</a>).</p>
-            <h3>Directions</h3>
             <ul>
-              <li>From Arlanda Airport, take bus or train to Stockholm C, and then the subway as described below.</li>
-              <li>From Bromma Airport, take bus to Stockholm C, and then the subway as described below.</li>
-              <li>From Stockholm C / T-Centralen, take the <b>red</b> subway line 14 to the station "Tekniska H&ouml;gskolan"</li>
+              <li><a href="https://us.pycon.org/2014/speaking/proposal_advice/">Proposal advice</a></li>
+              <li><a href="https://us.pycon.org/2014/speaking/proposal-resources/">Proposal resources</a></li>
+              <li><a href="https://us.pycon.org/2014/speaking/proposal_advice/samples/SpacePug/">Proposal sample</a></li>
```

Comment:
There are _2015_ sites for all those PyCon-proposal help pages, I think we should use them instead.